nautobot_golden_config/tests/test_views.py

```diff
@@ -350,3 +351,69 @@ def setUpTestData(cls):
     @skip("TODO: 2.0 Figure out how to have pass.")
     def test_list_objects_with_constrained_permission(self):
         pass
+
+
+@override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+class ConfigComplianceUIViewSetTestCase(
+    ViewTestCases.BulkDeleteObjectsViewTestCase,
+    # ViewTestCases.ListObjectsViewTestCase,  # generic list view tests won't work for this view since the queryset is pivoted
+):
+    """Test ConfigComplianceUIViewSet views."""
+
+    model = models.ConfigCompliance
+
+    @classmethod
+    def setUpTestData(cls):
+        create_device_data()
+        dev01 = Device.objects.get(name="Device 1")
+        dev02 = Device.objects.get(name="Device 2")
+        dev03 = Device.objects.get(name="Device 3")
+        dev04 = Device.objects.get(name="Device 4")
+
+        for i in range(4):
+            feature_dev01 = create_feature_rule_json(dev01, feature=f"TestFeature{i}")
+            feature_dev02 = create_feature_rule_json(dev02, feature=f"TestFeature{i}")
+            feature_dev03 = create_feature_rule_json(dev03, feature=f"TestFeature{i}")
+
+            updates = [
+                {"device": dev01, "feature": feature_dev01},
+                {"device": dev02, "feature": feature_dev02},
+                {"device": dev03, "feature": feature_dev03},
+                {"device": dev04, "feature": feature_dev01},
+            ]
+            for i, update in enumerate(updates):
```

Comment:
Perhaps? and the following`i`s
```suggestion
            for j, update in enumerate(updates):
```